thorlcr/master/thgraphmanager.cpp

```diff
@@ -327,13 +327,16 @@ void CJobManager::fatal(IException *e)
 
 void CJobManager::updateWorkUnitLog(IWorkUnit &workunit)
 {
-    StringBuffer log, logUrl;
+    StringBuffer log, logUrl, slaveLogPattern;
     logHandler->getLogName(log);
     createUNCFilename(log, logUrl, false);
-    workunit.addProcess("Thor", globals->queryProp("@name"), 0, logUrl.str());
-}
-
 
+    //Not sure whether this is the correct way to set the slaveLogPattern or not:
+    slaveLogPattern.set("thorslave.").append(SLAVEIDSTR);
+    slaveLogPattern.append(strstr(log, "/thormaster.") + strlen("/thormaster"));
```

Comment:
not ideal burning in "thormaster" here and having to make sure matches in workunit code.
Ideally should be a constant defined somewhere, included and used.
But I'm not sure where is best, not jlib (I would also not put SLAVEIDSTR there).

Reply:
How about '#define MASTERLOGSEARCHSTR "/thormaster"' in thorhelper.hpp?

Reply:
yes that may be fine.